formatter/fmt

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/env bash
+
+set -e
+set +x
+set -o pipefail
+
+cd /code
+
+# Gets the list of file paths for files that are tracked by git and have
+# changes relative to the main branch.
+function get_tracked_modified_files() {
+  git diff --name-only main
+}
+
+# Gets the list of file paths for files that are not tracked by git and are
+# not excluded by the .gitignore file.
+function get_untracked_files() {
+  git ls-files --others --exclude-standard
+}
+
+function get_modified_java_files() {
+  (
+    get_tracked_modified_files
+    get_untracked_files
+  ) | grep -E "server\/.+\.java"
+}
+
+function get_modified_ts_files() {
+  (
+    get_tracked_modified_files
+    get_untracked_files
+  ) | grep -E ".+\.ts"
```

Comment:
Prettier modifies more than just .ts files (including json, js, yaml, toml, ts, html and more: https://prettier.io/docs/en/)

Also, I _ think _ prettier is pretty quick.  We might get 90% of the speedup just from java.